models/tmp/stg_shopify__order_tmp.sql

```diff
@@ -1,6 +1,6 @@
 {{
     fivetran_utils.union_data(
-        table_identifier='order', 
+        table_identifier='order',
```

Comment:
@fivetran-reneeli actually does this need to be `shopify_order_identifier` ?

Reply:
sweet will also update

Reply:
wait wait i'm wrong!!!

Reply:
it should be the name of the source table node (which we ourselves have defined), _not_ the source table node's identifier